### ISSUE TITLE
Help and configuring YOLO correctly ?

### ISSUE THREAD

**Issue by newinai** [qqwweee/keras-yolo3#415]
Hi,

I have some questions about YOLO configuration for my AI problem.

**To resume :**
- 2 classes to detect in my dataset, A with 4500 features and B with 1900 features.
- A shapes are generaly big and B shapes smaller (depends of the picture).
- Original pictures are 6000x4000px.

**Configuration:**
- 2 GPU.
- batch_size=16.
- epochs=1000
- size: 416x416

**After multiple training trials :**
-  losses are stuck :  loss ~29 ; val_loss ~31

**Evaluation :**
- mAP is really bad ~ < 1

**Some questions :**
- Do i need to calculate my own dataset anchors ? If yes, is it compatible with my original size 6000px*4000px ?
- Is there a relation between anchors and 416x416 size ?
- Do i need to increase 416x416 size ?
- Why my loss / val_loss not decreasing more ?
- What do you suggest more ?

Thank you in advance !

**Comment by robisen1** [comment]
Are your classes very different from the COCO classes? When you say original size do you mean you cut the parts of interest out of the large image, or did you rescale the image, or you are passing that whole image to the system? I am assuming you are not passing the whole image since that would take a long time to train. Also are the images very similar meaning the same object in the same place at maybe a slight change in direction, distance, etc? Otherwise, everything looks like your doing it right.
